src/ralph/publishers.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
```

Comment:
imo `ralph/publishers.py` is not the best place for this. Could it be:
* defined not in main scope of ralph?
* gather all signals in one place?

Maybe new app for such kind of things?